src/rpm_ostree/cli_status.rs

```diff
@@ -170,41 +159,26 @@ fn status_json(client: &mut RpmOstreeClient) -> Fallible<StatusJSON> {
 }
 
 /// CLI executor for `rpm-ostree status --json`.
-pub fn invoke_cli_status(booted_only: bool) -> Fallible<StatusJSON> {
+pub fn invoke_cli_status() -> Fallible<rpmostree_client::Status> {
     RPM_OSTREE_STATUS_ATTEMPTS.inc();
 
-    let mut cmd = std::process::Command::new("rpm-ostree");
-    cmd.arg("status").env("RPMOSTREE_CLIENT_ID", "zincati");
```

Comment:
We lose this client ID - need to add that to upstream API.  It doesn't matter much for status querying but it will for other work.